frontend/src/components/common/Resource.tsx

```diff
@@ -331,23 +332,32 @@ export function SectionGrid(props: SectionGridProps) {
 
 export function DataField(props: TextFieldProps) {
   const { label, value, ...other } = props;
+  let language = ((label as string).split('.').pop() as string);
```

Comment:
I think the changes to the DataField are not related to changing the monaco editor, so they should be done in a different commit that explains why we're doing this change (the actual prettify purpose).